Monika After Story/game/script-topics.rpy

```diff
@@ -8960,6 +8960,217 @@ label monika_hydration:
+            m "Although I shouldn't be the one talking about that. After all...I'm the one having my eyes on you all the time~"
+            m 5ekbfu "Ehehe~ I love you so much, [player]."
+    return
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_planets",category=['misc'],prompt="Planets",random=True))
+
+label monika_planets:
+    m 1lua "Have you ever wondered how lucky we all are?"
+    m 4esa "If you think about it, the possibilities of having an existing planet that is suitable for human life are quite sparse."
+    m 4eka "Most of the celestial bodies out there don't even support any kind of human activity."
+    m 4eku "Some of them have little to no gravity, no oxygen, or not even terrain were to stand on!"
+    m 1eku "Yet, here we are, in planet Earth."
+    m 1hub "Isn't that great? You should be feeling lucky right now just for being alive!"
+    m 2euu "It's almost like life itself gave us the oportunity to live..."
+    m 3esb "You better remember that next time you are feeling down, [player]!"
+    m 3tsb "Or... You could also come to me and allow me to lift your mood~"
+    m 1hublb "Ehehe~"
+    return
+
+define persistent._mas_didplayerbetray = None
+
+init 5 python:
+   addEvent(Event(persistent.event_database,eventlabel="monika_betrayal",category=['misc'],prompt="Betrayal",random=True))
+
+label monika_betrayal:
+    m 1eka "...Say, [player]?"
+    m 1lka "Something came to my mind right now."
+    m 1eka "Have you ever betrayed someone by any means?"
+    m 4eka "I don't mean to 'betray' little promises with friends that could even be considered jokes."
+    m 4eku "I mean it in a real way... You know, breaking the confidence someone had with you because you did something terrible to them."
+    m "You don't have to answer if you don't want to, it's completely understandable."
+    $ _history_list.pop()
+    menu:
+        m "You don't have to answer if you don't want to, it's completely understandable."
+        "Yes, I have betrayed someone.":
+            $ persistent._mas_didplayerbetray = True
+            m 2ekc "..."
+            m "It's alright, [player]"
+            m 3ekc "I know you probably didn't mean to do that."
+            m 3ekb "We all mess up sometimes, don't we?"
+            m 3esb "It's not like I'm going to hate you for what you did, you silly~"
+
+        "No, I haven't betrayed anyone.":
+            $ persistent._mas_didplayerbetray = False
+            m 2huu "I'm happy to hear that, [player]."
+            m 3efb "People's trust isn't something to be played with!"
+            m 3esa "And just for letting you know..."
+            m 3hublb "I...{w=2}will never betray you!~"
+
+        "...":
+            m 2ekc "It's alright, [player]."
+            m "I won't force you to say anything."
+            m 3ekc "What kind of girlfriend would I be if I did that?"
+            m 3ekb "Ehehehe...Let's just forget this topic existed in first place."
```

Comment:
I think this line is too light for the mood. 

We probably want to say something other than "topic", especially since it looks like she's referring to the actual selectable topic itself.

Reply:
Maybe if I erase the "Ehehehe..." part and change "topic" to conversation. Could that be better?
Also changed part of the conversation to make it deeper and more serious.